src/app/community/create/create.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-community-create',
+  templateUrl: './create.component.html',
+  styleUrls: ['./create.component.scss']
+})
+export class CreateComponent implements OnInit {
+  nameText = '';
+  titleText = '';
+  descriptionText = '';
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+  nameFormControl = new FormControl('', [
```

Comment:
If you move this and `titleFormControl` above `constructor()`, it'll fix your linting error. Object instance properties should be declared before instance methods.

Reply:
ooh I see. Whoops. Thank you!